actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add `:allow_other_host` option to `redirect_back` method.
+    When `allow_other_host: false` redirection back to URLs from
+    a different host is not allowed. `allow_other_host` is `true` by default.
```

Comment:
I'm having trouble parsing what this sentence is saying "When `allow_other_host: false` redirection back to URLs from a different host is not allowed". Should it say something more like "When `allow_other_host` is set to false, the `redirect_back` URL will now allow a redirecting from a different host."

Reply:
What about:
"When `allow_other_host` is set to `false`, the `redirect_back` will not allow a redirecting to a different host."

Reply:
Yup that's better.

Reply:
Amended